create team management section of information gathering page. (#91)
wireapp/wire-docs: src/how-to/install/ansible-dependencies.rst

Dependencies on operator's machine
----------------------------------

We provide a container containing a compatible software stack (binaries)
required to run the ansible playbooks, helm charts etc.

If you don't intend to develop *on the tooling itself*, you should use this.

Use the provided Docker container
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> On your machine, you need to have `docker` installed (or any other Container

Author: ```suggestion
On your machine, you need to have `docker` installed (or any other compatible
```

> runtime really, even though instructions may differ).

Author: ```suggestion
container runtime. Instructions may differ).
```

See `how to install docker <https://docker.com>`__. Then:

::

>    docker pull quay.io/wire/wire-server-deploy-deps:latest

Reviewer: Perhaps we should use `master` and `develop` labels to signify we're tracking those branches?

>    # make sure the server accepts your ssh key for user root

Author: ```suggestion
   # Copy your SSH key to the hosts you are going to deploy wire onto
```

>    ssh-copy-id -i ../dot_ssh/id_ed25519.pub root@<server>

Author: all of this ssh manipulation assumes root, which is again, not required.

Author: additionally, we have not told the user to run an ssh agent.

Reviewer 2: ssh agents don't work, see the comment in the ansible inventory.

>    ssh-copy-id -i ../dot_ssh/id_ed25519.pub root@<server>

Author: ```suggestion
   ssh-copy-id -i ../dot_ssh/id_ed25519.pub <user>@<server>
```

   docker run -it --network=host -v $(pwd):/mnt -v $(pwd)/../dot_ssh:/root/.ssh -v $(pwd)/../dot_kube:/root/.kube quay.io/wire/wire-server-deploy-deps:latest
   # inside the container, copy everything to the mounted host file system:
   cp -a /src/* /mnt
   # and make sure the git repos are up to date:
   cd /mnt/wire-server && git pull
   cd /mnt/wire-server-deploy && git pull
   cd /mnt/wire-server-deploy-networkless && git pull

Now exit the docker container.  On subsequent times:

::

   cd admin_work_dir

>    docker run -it --network=host -v $(pwd):/mnt -v $(pwd)/../dot_ssh:/root/.ssh -v $(pwd)/../dot_kube:/root/.kube quay.io/wire/wire-server-deploy-deps:latest

Author: This all assumes one is running as root. There is no reason for this assumption, and most users have used the 'ubuntu' user for these steps.

::

   docker exec -it `docker ps -q --filter="ancestor=quay.io/wire/wire-server-deploy-deps:latest"` /bin/bash

Use Nix and Direnv to provide dependencies
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> We use a custom version of `ansible` (with some additional python

Reviewer 3: Why even mention this? This is all transparant to the user if they use the docker container; no?

> We use a custom version of `ansible` (with some additional python

Author: ```suggestion
We use a specific version of `ansible` (with some additional python
```

dependencies), a specific version of `terraform`, `helm`, `kubectl`, `gnumake`
and some more tooling.

These are all built via `Nix <https://nixos.org>_`, and added to `$PATH` by
`Direnv <https://direnv.net/>_`, so you would need these two things installed
and setup to make use of it - but if you just want to use the tools, see for
the instructions above.

Download external Ansible Roles
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

We use git submodules to manage external ansible roles.

> We recommend setting `git config --global submodule.recurse true` and `git

Reviewer 3: See my comment here: https://github.com/wireapp/wire-server-deploy/pull/363#issuecomment-716380427
